release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2396,6 +2396,116 @@ $ ./openshift-install wait-for install-complete
+The workaround is to create the subscription by calling the Redfish API directly, for example, by running the following command:
++
+[source,terminal]
+----
+curl -X POST -i --insecure -u "<BMC_username>:<BMC_password>" https://<BMC_IP>/redfish/v1/EventService/Subscriptions \
+    -H 'Content-Type: application/json' \
+    --data-raw '{
+    "Protocol": "Redfish",
+    "Context": "any string is valid",
+    "Destination": "https://hw-event-proxy-openshift-bare-metal-events.apps.example.com/webhook",
+    "EventTypes": ["Alert"]
+    }'
+----
++
+You should receive a `201 Created` response and a header with `Location: /redfish/v1/EventService/Subscriptions/<sub_id>` that indicates that the Redfish events subscription is successfully created. 
+(link:https://issues.redhat.com/browse/OCPBUGSM-43707[*OCPBUGSM-43707*])
+
+* When using the GitOps ZTP pipeline to install a {sno} cluster in a disconnected environment, there should be two `CatalogSource` CRs applied in the cluster. One of the `CatalogSource` CRs gets deleted following multiple node reboots. As a workaround, you can change the default names, such as `certified-operators` and `redhat-operators`, of the catalog sources. (link:https://issues.redhat.com/browse/OCPBUGSM-46245[*OCPBUGSM-46245*])
+
+* If an invalid subscription channel is specified in the subscription policy that is used to perform a cluster upgrade, the {cgu-operator-full} indicates a successful upgrade right after the policy is enforced because the `Subscription` state remains `AtLatestKnown`. (link:https://issues.redhat.com/browse/OCPBUGSM-43618[*OCPBUGSM-43618*])
+
+* The `SiteConfig` disk partition definition fails when applied to multiple nodes in a cluster. When a `SiteConfig` CR is used to provision a compact cluster, creating a valid `diskPartition` config on multiple nodes fails with a Kustomize plug-in error. (link:https://issues.redhat.com/browse/OCPBUGSM-44403[*OCPBUGSM-44403*])
+
+* If secure boot is currently disabled and you try to enable it using ZTP, the cluster installation does not start. When secure boot is enabled through ZTP, the boot options are configured before the virtual CD is attached. Therefore, the first boot from the existing hard disk has the secure boot turned on. The cluster installation gets stuck because the system never boots from the CD. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])
+
+* Using {rh-rhacm-first}, spoke cluster deployments on Dell PowerEdge R640 servers are blocked when the virtual media does not disconnect the ISO in the iDRAC console after writing the image to the disk. As a workaround, disconnect the ISO manually through the Virtual Media tab in the iDRAC console. 
+(link:https://issues.redhat.com/browse/OCPBUGSM-45884[*OCPBUGSM-45884*])
+
+* Low-latency applications that rely on high-resolution timers to wake up their threads may very infrequently experience higher wake up latencies than expected. 
```

Comment:
s/may/might or can

I would likely remove _very infrequently_ as it is vague and undefined.

Reply:
done